### DISCUSSION TITLE
Is polly works with gRPC?

### DISCUSSION THREAD

**Discussion by soroshsabz** [App-vNext/Polly#1801] [Q&A]
Does any extensions and specialization for gRPC exist? (specially for gRPC code first)

**Comment by martincostello** [comment]
I'm not aware of any gRPC specific extensions for Polly myself, certainly not here in the main repository. Polly is generally agnostic to the scenario the resilience is being used in (HTTP, SQL, etc).

**Reply by soroshsabz** [reply]
Do you know any docs exist for extend polly for support something like grpc or custom tcp socket in .NET 8 ASP.NET based projet?

thanks

**Reply by soroshsabz** [reply]
@martincostello

**Reply by martincostello** [reply]
No I don't.

**Reply by soroshsabz** [reply]
Thanks for responding, so Do you know any open source project or library to extend polly for supporting another communication channel, like grpc or something else, to I can see it, to help me to follow it?

thanks

**Reply by martincostello** [reply]
The only thing I'm aware of is that it doesn't work: https://github.com/dotnet/extensions/issues/4923
